types/stripe-js/elements/payment.d.ts

```diff
@@ -297,3 +322,112 @@ export interface StripePaymentElementChangeEvent {
     };
   };
 }
+
+type CardBrand =
```

Comment:
We have [CardNetworkBrand](https://github.com/stripe/stripe-js/blob/906a44c5/types/stripe-js/elements-group.d.ts#L597C13-L597C29) already, would it make sense to use that instead? Also cartes_bancaires and a few others from `CardNetworkBrand` are missing here.

Reply:
This is on purpose. We are only exposing the card brands that are publicly documented in the `payment_method` card object: https://docs.google.com/document/d/1dNA70BIrlKuXw02nmTpkiRUMApxSKjaVeyVjI1vmvBQ/edit?pli=1#heading=h.bhsr8nwiiq7d

I'm surprised we're exposing those brands in CardNetworkBrand. There are types that the card-metadata service returns that aren't publicly documented